### ISSUE TITLE
Buttons 13 and 14 still do their original actions

### ISSUE THREAD

**Issue by lostallmymoney** [RaulPPelaez/Naga_KeypadMapper#67]
On my naga 2014 they still do backward and forward except if i run
'ioctl(extra_btn_fd, EVIOCGRAB, 1);'
but then all buttons are disabled. Thanks for helping me fix this.

**Comment by RaulPPelaez** [comment]
What do you mean by "all buttons are disabled"?

Anyway you should probably try to shut down the extra buttons using xinput instead of doing it in naga.cpp.
This is done with a line similar to this one in nagastart.sh:
```
xinput set-int-prop $NAGAID1 "Device Enabled" 8 0 
```
running xinput will give you clues on what the device id is.

**Comment by lostallmymoney** [comment]
The mouse left click, right click and cursor movements get disabled because of ioctl. Thanks for the xinput command I'll try that rn

**Comment by lostallmymoney** [comment]
Yeah I can do the same thing as ioctl with xinput but it also disable the usual clicks of the mouse. :
xinput set-int-prop 18 "Device Enabled" 8 0
gets my 2 extra buttons disabled but the clicks and mouse movements are also disabled.

**Comment by lostallmymoney** [comment]
So the fix is to add 275 and 276 in xinput button maps

**Comment by RaulPPelaez** [comment]
If I am understanding the click and top buttons are part of the same device so you cannot disable it.
Then you should play around with xinput set-button-map and make these top buttons do nothing instead of disabling the device. Maybe you can xinput set-button-map the top buttons to 0, which I think means no button.
